mne/viz/epochs.py

```diff
@@ -413,6 +415,24 @@ def plot_epochs(epochs, picks=None, scalings=None, n_epochs=20,
     title : str | None
         The title of the window. If None, epochs name will be displayed.
         Defaults to None.
+    events : None, array, shape (n_events, 3)
+        Events to show with vertical bars. If events are provided, the epoch
+        numbering is off by default to prevent overlap. You can toggle epoch
```

Comment:
just a nitpick = instead of "epoch numbering is off by default to prevent overlap" it might be a bit clearer to say that "epoch numbers are not shown to prevent overlap"